less/site.less

```diff
@@ -4,8 +4,19 @@
 @import 'util.less';
 
 :root {
-	--link-color: #73acfe;
-	--logo-color: #73acfe;
+	--black: #191a25;
+	--offblack: #292A3C;
+	--offwhite: #F7F7F5;
+	--fifth--secondary-shadow: #d2d219;
+	--fifth-color: #f0f00a;
+	--fourth--secondary-shadow: #6114a3;
+	--fourth-color: #7a21c5;
+	--third--secondary-shadow: #01b6ae;
+	--third-color: #62d6d2;
+	--secondary-shadow: #E9513F;
+	--secondary-color: #ff5945;
+	--primary-shadow: #F0C106;
+	--primary-color: #ffdc00;
 }
```

Comment:
What do these names mean? I don't get the `fifth` and `fourth`.

Reply:
brand colors primary, secondary, third could tertiary I guess, but I don't know how to say fourth and fifth in the same way. quartiary and quintessential? 😄

Reply:
That's how I've always done brand colors, any suggestions?

Reply:
I prefer calling them a number in hierarchy rather than the name of the color, because that way I change a brand color from purple to green and still be the 5th brand color.

Reply:
ah, thanks for clarifying.